tests/test_transpile.py

```diff
@@ -747,7 +747,6 @@ def test_command_identity(self):
             "ALTER SEQUENCE IF EXISTS baz RESTART WITH boo",
             "ALTER SESSION SET STATEMENT_TIMEOUT_IN_SECONDS=3",
             "ALTER TABLE integers DROP PRIMARY KEY",
-            "ALTER TABLE s_ut ADD CONSTRAINT s_ut_uq UNIQUE hajo",
```

Comment:
Removed this test case from here, as this syntax is now supported (and this query no longer gets parsed as an `exp.Command` but as an `exp.AlterTable` expression now). Please confirm if this change is appropriate @georgesittas

Reply:
Yep, this change looks reasonable. Thanks for double-checking it.